packages/api-core/src/site-activity-log/types.ts

```diff
@@ -26,12 +27,95 @@ export interface ActivityLogEntry {
 	};
 	published: string;
 	rewind_id: string;
+	status: ActivityStatus;
 	summary: string;
 	streams: ActivityLogEntry[];
 }
+export interface ActivityLogIcon {
+	type: 'Image';
+	url: string;
+	width: number;
+	height: number;
+}
+export interface ActivityNotificationRange {
+	// Node-local unique ID and parent reference
+	id: string;
+	parent?: string | null;
+	// UCS-2 indices within `text` [start, end)
+	indices: [ number, number ];
+	// Optional presentational attributes
+	style?: string;
+	class?: string;
+	section?: string;
+	url?: string;
+	type: string;
+	intent?: string;
+}
 
-export interface ActivityLog {
+export interface ActivityLogAPIResponse {
```

Comment:
super nit: Let's call it `ActivityLogResponse`. We don't seem to have this `*APIResponse` suffix anywhere else in the package; only `*Response`.

Reply:
@fushar I actually took it from the site logs https://github.com/Automattic/wp-calypso/blob/daab2961a214e063c70b1a4fecc8dd52db19b18e/packages/api-core/src/site-logs/types.ts

Happy to refactor it without the API part if you think it's better.